engine/resources/exec.go

```diff
@@ -800,3 +821,21 @@ func (obj *wrapWriter) Write(p []byte) (int, error) {
 func (obj *wrapWriter) String() string {
 	return obj.Buffer.String()
 }
+
+// checks that environment variable name is valid
+func isNameValid(varName string) error {
+	if varName == "" {
+		return fmt.Errorf("variable name cannot be an empty string")
+	}
+	i := 0
```

Comment:
This line and the next can be combined. Know how?

Reply:
I believe lines 830 and 831 can be combined like so:
```
for i := 0; i < len(varName); i++ {
```